plugin.json

```diff
@@ -15,7 +15,7 @@
         "bundle_path": "webapp/dist/main.js"
     },
     "settings_schema": {
-        "header": "To set up the GitHub plugin, [register a GitHub OAuth app](https://github.com/settings/applications/new) with `https://your-mattermost-url.com/plugins/github/oauth/complete` as your Authorization callback URL.",
+        "header": "The Github plugin for Mattermost allows users to get Daily reminders, Notifications, Sidebar buttons, and Slash commands - for common interations with Github directly from within Mattermost. \n \n To get started, you'll need to [register a GitHub OAuth app](https://github.com/settings/applications/new) with `https://your-mattermost-url.com/plugins/github/oauth/complete` as your Authorization callback URL. \n \n Continue to follow the [setup instructions here](https://www.mattermost.com/pl/default-github-plugin#configuration)",
```

Comment:
This is great! Propose
 - Github > GitHub
 - "Sidebar buttons" or other such features may not be obvious at first, so we could generalize to use cases instead in the first sentence, e.g. `Subscribe to notifications, stay up-to-date with reviews ...`

Reply:
Sorry, I copy pasted from an old version!